### ISSUE TITLE
[Visual Requirements – Make Code – Home Page] Content present on titles are not properly visible after resizing at 400% under ‘Home Page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3442]
**User Experience:**
End user will find difficulty to interact with controls and won't be able to perform functionality over such control if controls are not visible after resizing to 400% due to this, user will be affected.

**Note:** User credentials should NOT be included in the bug.

**Re-pro  Steps:**
1. Launch application URL : https://makecode.microbit.org/# 
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Go to browser setting page Zoom up to 400%.
5. Verify the issue.
 
**Actual Result:**
Content present on titles are not properly visible after resizing at 400% under ‘Home Page’, due to this, user is unable to interact with controls and won't be able to perform functionality over such controls.
 
**Expected Result:**
Content present on titles should be properly visible after resizing at 400% under ‘Home Page’, so that user could easily interact with controls and could be able to perform functionality over such controls.

**Note:**
Same issue exists on Chrome & Firefox.

**MAS Reference:**
[MAS 1.4.10 - Reflow
](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BE12828DF-B35C-4F4D-9289-1F932E19F8CF%7D&file=MAS%201.4.10%20%E2%80%93%20Reflow.docx&action=default&mobileredirect=true&cid=41e61a50-333f-40f8-9f0a-95240dd6fc63)

**Reference Links:**
•	[Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
•	External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
•	Please reach out to [C&AI Teams channel](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fteam%2F19%3A3cd071821b694f11b054d878453cd75f%40thread.skype%2Fconversations%3FgroupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&type=team&deeplinkId=99c73412-3928-459b-a051-ad90788e6969&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true) for any process related queries.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA
#URL:https://makecode.microbit.org/#
